src/action.i.ts

```diff
@@ -0,0 +1,3 @@
+export default interface Action {
+  execute(title: string, args: string[], outs: any[], context: any): Promise<string[]>;
```

Comment:
Whatever the `execute` method returns goes as `outs`.

Reply:
Done